Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -462,6 +472,33 @@ function Add-RegistrationApiRegistration([string] $identityServerUrl, [string] $
     return $registrationApiSecret
 }
 
+function Add-IdpssApiResourceRegistration($identityServiceUrl, $fabricInstallerSecret)
+{
+    $accessToken = Get-AccessToken -authUrl $identityServiceUrl -clientId "fabric-installer" -secret $fabricInstallerSecret -scope fabric/identity.manageresources
+
+    Write-DosMessage -Level Information "Registering IdentitySearchProvider API with Fabric.Identity..."
+    $apiName = "identitySearchProvider-service-api"
+    try{
+        [string]$apiSecret = [string]::Empty
+        Write-Host "    Registering $($apiName) with Fabric.Identity"
+        $body = New-APIRegistrationBody -apiName $apiName -userClaims @("name", "email", "roles", "group") -scopes @{"name" = "fabric/idprovider.searchusers"} -isEnabled $true
+        $apiSecret = New-ApiRegistration -identityUrl $identityServiceUrl -body (ConvertTo-Json $body) -accessToken $accessToken
+
+        if (![string]::IsNullOrEmpty($apiSecret) -and ![string]::IsNullOrWhiteSpace($apiSecret)) {
```

Comment:
I think IsNullOrWhiteSpace is sufficient here, It also does a null check.

Reply:
yup.